opentelemetry-sdk/src/opentelemetry/sdk/metrics/_internal/aggregation.py

```diff
@@ -126,17 +126,17 @@ def __init__(
         )
         self._instrument_is_monotonic = instrument_is_monotonic
 
-        self._current_value = None
```

Comment:
I renamed `_current_value` to `_value` and `_previous_cumulative_value` to `_previous_value` for better consistency.